### PR TITLE
Pull selected services definitions for testing

### LINKED ISSUE #867: [Tests] remove `eris init`'s and `eris clean`'s from tests_tooltests_jenkins
**Problem** -- our unit tests can be more cleanly isolated. The test runner for those jobs should be as minimalistic as possible. Indeed, it should mostly be controlling `go tests` and that's it. 

**Solutions** -- a good start will be removing the eris inits and eris cleans from the file noted above. We'll need some deeper refactors as well.

### PR DESCRIPTION
**Pull request by pietv**
* `tests.TestsInit()` takes additional arguments to pull selected services:
  
  ```
  ...
  tests.TestsInit(ConnectAndPull, "keys", "ipfs", "do_not_use")
  ...
  ``` 
* Renames `files.go` and `files_arm.go` definitions not to include the `.toml` suffix (looks better/dries out both `tests.TestsInit()` function and the definitions list).
* Includes "ipfs" and "keys" back into `SERVICE_DEFINITION` to optimize pulling of the custom list.
* Partially resolves #867.

### CODE REVIEW

File: initialize/initialize_test.go [hunk 1]
```diff
@@ -97,12 +98,12 @@ func testDrops(dir, kind string) error {
 	case "services":
 		//pull from toadserver
 		if toadUp {
-			if err := dropServiceDefaults(dirToad, "toadserver"); err != nil {
+			if err := dropServiceDefaults(dirToad, "toadserver", ver.SERVICE_DEFINITIONS); err != nil {
```
Comment (zramsay) [inline initialize/initialize_test.go]: all references to toadserver can be removed. that's (mostly) handled in #913 I'll rebase that PR when this oneis merged.

File: services/services_test.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ func TestMain(m *testing.M) {
 	// log.SetLevel(log.InfoLevel)
 	// log.SetLevel(log.DebugLevel)
 
-	tests.IfExit(tests.TestsInit(tests.ConnectAndPull))
+	tests.IfExit(tests.TestsInit(tests.ConnectAndPull, "do_not_use", "keys", "ipfs"))
```
Comment (zramsay) [inline services/services_test.go]: should we eliminate `do_not_use` ?

**Comment by zramsay** [comment]
LGTM. comments do not affect PR in it's current form

File: services/services_test.go (hunk 2 above)
Comment (pietv) [inline services/services_test.go]: didn't know what it's used for in the first place. yeah, let's drop it!

Comment (zramsay) [inline services/services_test.go]: used for testing....added to #917

**Pull request merged**